JamitFoundation.podspec

```diff
@@ -0,0 +1,128 @@
+#
+#  Be sure to run `pod spec lint JamitFoundation.podspec' to ensure this is a
+#  valid spec and to remove all comments including this before submitting the spec.
```

Comment:
This comments says that you should remove all comments in the podspec file

Reply:
I removed the comments